app/models/attachable.rb

```diff
@@ -60,6 +66,35 @@ def attachables
     [self]
   end
 
+  def has_attachment_changes?
+    has_created_attachments? || has_updated_attachments? || has_deleted_attachments?
+  end
+
+  def created_attachments
+    attachments.select { |attachment| attachment.created_at > (created_at + 5.seconds) }
+  end
+
+  def updated_attachments
+    updated_attachments = attachments.select do |attachment|
+      attachment.updated_at > attachment.created_at ||
+        (attachment.respond_to?(:govspeak_content) &&
+        attachment.govspeak_content.updated_at > attachment.govspeak_content.created_at)
+    end
+    updated_attachments - created_attachments
+  end
```

Comment:
Also: I feel like we should probably do a couple of things with that `5.seconds`:

1. Sense check to make sure this is a reasonable delay to expect when new Editions are created. We have some documents with _hundreds_ of attachments, so I'd be interested to see if this `5.seconds` value works there.
2. Document it somehow to explain what this 'magic value' represents, why it's needed, etc. to avoid any future WTFs. An inline comment might do the trick. It might even be worth naming it as a named constant – although that might be too much 🤷🏻